keyboards/skyloong/gk104/q1/matrix.c

```diff
@@ -0,0 +1,154 @@
+// Copyright 2023 NaturaZh (@NaturalZh)
+// SPDX-License-Identifier: GPL-2.0-or-later
+//Matrix read rows set col:DIODE_DIRECTION == ROW2COL
+
+#include "quantum.h"
+#include "debounce.h"
+
+#define ReadDelayTime 120
+#define ClockTime 15
```

Comment:
same here.  And if anything, these defines, if being used in multiple places should probably be moved to a header file, to keep in sync.